app/app.go

```diff
@@ -21,11 +19,6 @@ type App struct {
 }
 
 func New(cfg *config.Config) (*App, error) {
-	logFlags := log.Lshortfile
-	if cfg.IsDevelopment() {
-		logFlags |= log.LstdFlags
-	}
-
```

Comment:
No longer need stdlib log flags since we're using the fancy structured logging package